operator/v1/types_network.go

```diff
@@ -440,6 +440,10 @@ type OVNKubernetesConfig struct {
 	// cluster.
 	// +optional
 	IPsecConfig *IPsecConfig `json:"ipsecConfig,omitempty"`
+	// ipsecExtern enables external (NS) IPsec for the node, the configuration
+	// is done via nmstate operator.
+	// +optional
+	IPsecExtern *IPsecExtern `json:"ipsecExtern,omitempty"`
```

Comment:
we might consider making it a flag not config like
`IPsecHostEnable` (with true/false possible values)

not 100% sure what's the best practice here though